pkg/kubelet/apis/cri/services.go

```diff
@@ -43,6 +43,8 @@ type ContainerManager interface {
 	ListContainers(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error)
 	// ContainerStatus returns the status of the container.
 	ContainerStatus(containerID string) (*runtimeapi.ContainerStatus, error)
+	// UpdateContainerResources updates the cgroup resources for the container.
+	UpdateContainerResources(containerID string, resources *runtimeapi.LinuxContainerResources) error
```

Comment:
If in the future we end up updating any other properties of containers, we will have to introduce a new top level API. I was wondering if we can make this API generic and control what features can be updated via input options.